### PR TITLE
Fix humphd/brackets#190 - Add a button to toggle between an attached and detached live preview state

### PR DESCRIPTION
**Pull request by AJDBenner**
Fixes humphd/brackets#190. This PR adds a button for the live preview. It's a WIP and still requires the function call to open the detached live preview window. Since we're no longer just hiding UI preferences we are altering the UI, I changed the `hideUI.js` file to `UI.js` and the `hide()` function to `initUI()`.

This is what the change looks like.
screen shot 2015-04-15 at 2 14 05 pm

### CODE REVIEW

**Comment by AJDBenner** [comment]
Previous comments can be found [here](https://github.com/humphd/brackets-browser-livedev/pull/61)

**Comment by humphd** [comment]
Some options from http://modernuiicons.com/ might be:

appbar fullscreen box

appbar fullscreen

**Comment by AJDBenner** [comment]
What do you think of this?
screen shot 2015-04-15 at 3 18 45 pm

**Comment by humphd** [comment]
This needs to be rewritten to use the `StatusBar` API in Brackets so we properly register a section of the status bar, see https://github.com/adobe/brackets/blob/b12f8eda9cddc0571814e4e0257827e0bb31ef6d/src/language/CodeInspection.js#L653-L655 as one example.  Also, there is some odd vertical centering happening in that screenshot, your sizes might need some help.

**Comment by AJDBenner** [comment]
What do you think of the vertical centering look now?
screen shot 2015-04-15 at 4 26 32 pm

File: lib/UI.js [hunk 1]
```diff
@@ -6,18 +6,23 @@ define(function (require, exports, module) {
...
      * for bramble to be loaded
      */
-    function hide() {
+    function initUI() {
         if(shouldHideUI()) {
             removeMainToolBar();
             removeLeftSideToolBar();
             removeRightSideToolBar();
             removeTitleBar();
+            addLivePreviewButton();
```
Comment (humphd) [inline lib/UI.js]: Move this out of the `if` block, since we always want to do it.

File: lib/UI.js [hunk 2]
```diff
@@ -86,8 +91,26 @@ define(function (require, exports, module) {
...
+     * Used to add a button to the status bar for the
+     * detached live preview.
+     */
+    function addLivePreviewButton() {
+        var livePreview = Mustache.render("<a id='liveDevButton' href=#><img id='liveDevImg' src='extensions/default/brackets-browser-livedev/lib/LivePreviewButton.png'></a>", Strings);
+        StatusBar.addIndicator("liveDevButton", $(livePreview), true, "", "", "status-indent");
+
+        $("#liveDevImg").css({
+            "width": "30px", 
+            "height": "30px"
```
Comment (humphd) [inline lib/UI.js]: Let's do `25px`

**Comment by humphd** [comment]
r+ with changes I've indicated.

**Comment by AJDBenner** [comment]
Here's an updated picture of the button addition.

screen shot 2015-04-16 at 12 17 15 pm

**Comment by humphd** [comment]
r+, looks great!

**Pull request merged**